sdcm/tester.py

```diff
@@ -1692,7 +1716,7 @@ def get_nemesises_stats(self):
             for nem in self.db_cluster.nemesis:
                 nemesis_stats.update(nem.stats)
 
-        for name, detail in nemesis_stats.items():
+        for _, detail in nemesis_stats.items():
```

Comment:
`for _, detail in nemesis_stats.items():` -> `for detail in nemesis_stats.values():`